scripts/variantstore/docs/aou/vds/vds_validation.py

```diff
@@ -50,10 +50,8 @@ def main(vds):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(allow_abbrev=False, description='Create VAT inputs TSV')
-    parser.add_argument('--vds-path', type=str, help='Input VDS Path', default="@VDS_INPUT_PATH@",
-						required=True)
-    parser.add_argument('--temp-path', type=str, help='Path to temporary directory', default="@TEMP_DIR@",
-                        required=True)
+    parser.add_argument('--vds-path', type=str, help='Input VDS Path', required=True)
```

Comment:
TODO - WHY are there two copies of vds_validation.py??

Reply:
probably one was in the docker and one was created outside of that--let's definitely get rid of the one not being called in the docker image